tensorflow/python/keras/callbacks_test.py

```diff
@@ -725,12 +725,10 @@ def test_EarlyStopping_reuse(self):
           optimizer='sgd', loss='binary_crossentropy', metrics=['accuracy'])
       weights = model.get_weights()
 
-      stopper = keras.callbacks.EarlyStopping(monitor='acc', patience=patience)
```

Comment:
Why removing this block of assertion?

Reply:
Thanks @rchao, it may be duplicate, isn't it?